scripts/increment-cargo-version.sh

```diff
@@ -138,15 +136,6 @@ for Cargo_toml in "${Cargo_tomls[@]}"; do
   done
 done
 
-# Update all the documentation references
-for file in "${markdownFiles[@]}"; do
-  # Set new crate version
-  (
-    set -x
-    sed -i "$file" -e "s/$currentVersion/$newVersion/g"
-  )
-done
-
```

Comment:
I think we do still want this for some of the markdown files, like the one that produces the [cli-tool docs page](https://docs.solana.com/cli/install-solana-cli-tools), don't we? Or is that being handled somewhere else?

Reply:
Oh jk, I just read the discussion on discord (and clearly haven't actually looked at the code). Disregard this ignorant comment!